### PR TITLE
PSP-5175 : Create claims for projects

### PR DESCRIPTION
**Pull request by eddherrera**

### CODE REVIEW

File: source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql [hunk 1]
```diff
@@ -235,4 +235,31 @@ INSERT [dbo].[PIMS_CLAIM]
     ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
 VALUES 
     (42, N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'activity-delete', N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'Ability to delete activities.', 0, 1, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId)
-GO
\ No newline at end of file
+
+INSERT [dbo].[PIMS_CLAIM]
```
Comment (devinleighsmith) [inline source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql:239]: Hmm, didn't you say you were going to assign this to Doug?

**Comment by devinleighsmith** [comment]
@dfilteau please be aware of this change in your 45.01 PR. Do not overwrite these new claims, and make sure the build for 45.01 includes these claims.

**Review by devinleighsmith — APPROVED** [review]

File: source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql (hunk 1 above)
Comment (eddherrera) [reply source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql:239]: I am adding the Claim, but not the Role_Claims.  Should I just not add the claim either?

Comment (devinleighsmith) [reply source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql:234]: I would probably either do neither or both.

Comment (eddherrera) [reply source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql:234]: Removed altogether.

File: source/database/mssql/scripts/dbscripts/PSP_PIMS_S45_00/Alter Down/76_DML_PIMS_ROLE_CLAIM.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+DECLARE @projectAdd BIGINT;
+DECLARE @projectEdit BIGINT;
+DECLARE @projectDelete BIGINT;
+SELECT @projectView = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'project-view';
+SELECT @projectAdd = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'project-add';
+SELECT @projectEdit = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'project-edit';
+SELECT @projectDelete = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'project-delete';
+
+/* Functional */
+DELETE FROM [dbo].[PIMS_ROLE_CLAIM] WHERE CLAIM_ID = @projectView AND ROLE_ID = @functional;
```
[the remark this answers is not in the document]
Comment (eddherrera) [reply source/database/mssql/scripts/dbscripts/PSP_PIMS_S45_00/Alter Down/76_DML_PIMS_ROLE_CLAIM.sql]: Simplified.

**Pull request merged**